src/main/webapp/jsp/instructorStudentRecords.jsp

```diff
@@ -186,7 +184,7 @@
                                                     <div class="form-group form-inline">
                                                         <div class="form-group text-muted">
                                                             <p>
-                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name) %>:
+                                                                Comment about <%= InstructorStudentRecordsPageData.sanitizeForHtml(data.student.name + " (" + data.student.team + ", " + data.student.email + ")") %>:
                                                             </p>
```

Comment:
In Instructor Comments, it is only 'To Name (Team, Email)'. There is no 'Comment about ...'. The email is displayed once. Having it twice in records looks like duplication.